Fixed font-size rem default 16px (#61)

HappyViki opened the pull request:
Root font-size is 16px, rest font-size are rem. I used this [ref](http://www.standardista.com/px-to-rem-conversion-if-root-font-size-is-16px/) to convert.
All the rems that were not font-size have been changed to px.

joeldenning commented on frontend/about/about.component.js:
```diff
@@ -127,10 +127,10 @@ export const css = `
-    margin: ${rawNavbarHeight + 32}rem auto;
-    padding: 32rem;
-    border: 1rem solid ${darkGray};
+    max-width: 936px;
```
Oh no! I think there was a miscommunication. We **still want to use rems**, but just based on 1rem equaling 16px Sorry for the wasted work! All of your changes will need to be adjusted to be rems.

```suggestion
    max-width: 58.5rem;
```

joeldenning commented:
Please merge the master branch into your branch, which will fix the build (see related #85)

HappyViki commented:
I learned how to [copy from one branch to another](https://stackoverflow.com/questions/35174544/git-how-to-copy-contents-of-one-branch-to-other-branch#answer-35177459), woot! I learned about `font-size: 62.5%` from [this](https://medium.com/code-better/css-units-for-font-size-px-em-rem-79f7e592bb97) article.

Anyway, here's what I did:
```
:root { font-size: 62.5%; ... }
body { font-size: 1.6rem; ... }
```
And changed all previous `rem`s to what it was before divided by ten, example:
```
& button, & a.button {
  font-size: 18rem -> 1.8rem;
  border-radius: 6rem -> 0.6rem;
  ...
}
```
`font-size: 62.5%` in root makes it easier to translate default `px` -> `rem`, as you can see in the above example. If `font-size: 100%` in root, then you'd have to do something like `font-size: 18rem -> 11.25rem`, aka multiply by 62.5%.

Body font-size used to be 16px. I treated it like the other rems. You could say I did `16px` -> `16rem` -> `1.6rem`. If I removed that line, body `font-size` would be `16px * 62.5%` by default.

I hope I explained well. I did this mainly to learn from what I did today.
